src/components/editor/editor.tsx

```diff
@@ -12,7 +12,22 @@ import { TaskBar } from './task-bar/task-bar'
 
 const Editor: React.FC = () => {
   const editorMode: EditorMode = useSelector((state: ApplicationState) => state.editorConfig.editorMode)
-  const [markdownContent, setMarkdownContent] = useState('# Embedding demo\n\n## Slideshare\n{%slideshare mazlan1/internet-of-things-the-tip-of-an-iceberg %}\n\n## Gist\nhttps://gist.github.com/schacon/1\n\n## YouTube\nhttps://www.youtube.com/watch?v=KgMpKsp23yY\n\n## Vimeo\nhttps://vimeo.com/23237102')
+  const [markdownContent, setMarkdownContent] = useState('# Embedding demo\n' +
+    '\n' +
+    '## Slideshare\n' +
+    '{%slideshare mazlan1/internet-of-things-the-tip-of-an-iceberg %}\n' +
+    '\n' +
+    '## Gist\n' +
+    'https://gist.github.com/schacon/1\n' +
+    '\n' +
+    '## YouTube\n' +
+    'https://www.youtube.com/watch?v=KgMpKsp23yY\n' +
+    '\n' +
+    '## Vimeo\n' +
+    'https://vimeo.com/23237102\n' +
+    '\n' +
+    '## PDF\n' +
+    '{%pdf https://www.w3.org/WAI/ER/tests/xhtml/testfiles/resources/pdf/dummy.pdf %}')
```

Comment:
Can't we use template literals for this?

Reply:
done